### ISSUE TITLE
Training strategies & training code for LRS

### ISSUE THREAD

**Issue by xiaojieli0903** [ms-dot-k/Multi-head-Visual-Audio-Memory#2]
Thanks for sharing the code! Can you please share more implementation details for training on LRW to get the 88.5% accuracy on LRW test? For example, the initial learning rate, and the number of training epochs seem not listed in the paper and this project.  Also, can you release the training code for LRS2?

**Comment by ms-dot-k** [comment]
For the LRW training, we used an initial learning rate of 0.0001 and trained the network for about 400 epochs.
During training, we reduced the learning rate by half when there is no improvement in validation accuracy for 10 epochs. And we don't have a plan to release for the sentence-level lip reading code (LRS2) since it is related to our industrial project program.
